core/src/main/java/org/pentaho/cassandra/driver/datastax/DriverCQLRowHandler.java

```diff
@@ -152,22 +152,59 @@ public void newRowQuery( StepInterface requestingStep, String tableName, String
   }
 
   public static Object readValue( ValueMetaInterface meta, Row row, int i ) {
+    
+    if (row.isNull( i ) ) {
```

Comment:
So this will fail checkstyle right off.

if (row..... should be if ( row .....

Please make sure you run the site target in Maven to get the checkstyle output and then navigate to the target/site folder and open checkstyle.html to see all the violations in your PR. This line would fail.